satpy/demo/_zip.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Utilities for downloading and extracting demo data zip files."""
+
+import os
+import math
+import logging
+import requests
+from zipfile import ZipFile
+
+try:
+    from tqdm import tqdm
+except ImportError:
+    def tqdm(iter_content, **kwargs):
+        """Fake tqdm progress bar."""
+        return iter_content
+
+logger = logging.getLogger(__name__)
+
+
+def _download_data_zip(url, output_filename):
+    if os.path.isfile(output_filename):
+        logger.info("Data zip file already exists, won't re-download: {}".format(output_filename))
+        return True
+
+    print("Downloading {}".format(url))
```

Comment:
Why printing and not logging?

Reply:
So...this was copied from my half day tutorial where I just had it print stuff out to the user. It could use logging. I'll try to update that eventually.